utils/src/server/rpc_module.rs

```diff
@@ -404,14 +407,8 @@ impl SubscriptionSink {
 	pub fn close(&mut self, close_reason: String) {
 		self.is_connected.take();
 		if let Some((sink, _)) = self.subscribers.lock().remove(&self.uniq_sub) {
-			let result =
-				to_raw_value(&SubscriptionClosedError::from(close_reason)).expect("valid json infallible; qed");
-			let msg = serde_json::to_string(&JsonRpcSubscriptionResponse {
-				jsonrpc: TwoPointZero,
-				method: self.method,
-				params: JsonRpcNotificationParams { subscription: self.uniq_sub.sub_id, result: &*result },
-			})
-			.expect("valid json infallible; qed");
+			let msg =
+				self.build_message(&SubscriptionClosedError::from(close_reason)).expect("valid json infallible; qed");
```

Comment:
Whats `qed`?

Reply:
https://en.wikipedia.org/wiki/Q.E.D.

basically, hand-wavy proof that a given `unwrap/expect` is infallible

Reply:
It's latin, old-europe snobbish way of saying "tada!"